lib/free/set-status.js

```diff
@@ -12,7 +12,7 @@ function setStatusFree (newStatus, context) {
     head_sha: pullRequest.head.sha,
     status: 'in_progress',
     output: {
-      title: 'Work in progress',
+      title: `Title contains ${newStatus.match === '🚧' ? 'a construction sign!' : `"${newStatus.match}"`}`,
```

Comment:
This will only match the 🚧 emoji, but people can configure to use other emoji, it would be better to make that more flexible.

Maybe instead of "sign" we can say "emoji"?

Reply:
Right, forgot about that. 
To confirm I should check if it's an emoji & if it is then say `Title contains an emoji`? If it's not an emoji then display the text as per usual.

Also, if I'm not mistaken, this code should work in the free folder as free cannot set custom preferences.

Reply:
> Also, if I'm not mistaken, this code should work in the free folder as free cannot set custom preferences.

You are absolutely right, the check is good enough for the free plan :( 

> To confirm I should check if it's an emoji & if it is then say Title contains an emoji? If it's not an emoji then display the text as per usual.

Yes. Ideally we would show the name of the emoji, so instead of just saying "Title contains an emoji" we would say "Title contains a contruction emoji". I wonder if there is a package that can turns an emoji into its name :)